cohttp-eio/src/client.mli

```diff
@@ -24,6 +24,6 @@ val make :
     - URIs of the form "httpunix://unix-path/http-path" connect to the given
       Unix path. *)
 
-val make_generic : (sw:Switch.t -> Uri.t -> _ Eio.Net.stream_socket) -> t
+val make_generic : (sw:Switch.t -> Uri.t -> Eio.Flow.two_way_ty r) -> t
```

Comment:
```suggestion
val make_generic : (sw:Switch.t -> Uri.t -> _ Eio.Flow.two_way) -> t
```

Reply:
@talex5 Thanks for your feedback. I updated my patch.